examples/emodel_pipeline_local_python/optimisation.sh

```diff
@@ -1,12 +1,11 @@
 #!/bin/bash
 
-cd ..
 source ./myvenv/bin/activate
 
-export OPT_ETYPE="L5PC"
-export RESUME=0 # Put 0 to start a new run or "githash" too resume an optimisation
+export OPT_EMODEL="L5PC" # Your e-model name
+export RESUME="0" # Put 0 to start a new run or "githash" too resume an optimisation
```

Comment:
typo: too -> to

maybe say `a githash` instead of `"githash"` here. If a user is not very focus, s/he could believe that s/he is supposed to actually put the string `"githash"` here